corehq/apps/translations/integrations/transifex/views.py

```diff
@@ -296,6 +298,36 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
 
+@method_decorator([toggles.APP_TRANSLATIONS_WITH_TRANSIFEX.required_decorator()], name='dispatch')
+class BlacklistTranslations(BaseTranslationsView):
+    page_title = _('Blacklist Translations')
+    urlname = 'blacklist_translations'
+    template_name = 'blacklist_translations.html'
+    section_name = ugettext_noop("Translations")
+
+    def section_url(self):
+        return self.page_url
+
+    @property
+    def blacklist_form(self):
+        if self.request.POST:
+            return AddTransifexBlacklistForm(self.domain, self.request.POST)
+        return AddTransifexBlacklistForm(self.domain)
+
+    @property
+    def page_context(self):
+        context = super(BlacklistTranslations, self).page_context
+        context['blacklisted_translations'] = TransifexBlacklist.translations_with_app_name(self.domain)
+        context['blacklist_form'] = self.blacklist_form
+        return context
+
+    def post(self, request, *args, **kwargs):
+        if self.transifex_integration_enabled(request):
+            if self.blacklist_form.is_valid():
+                self.blacklist_form.save()
+        return self.get(request, *args, **kwargs)
```

Comment:
i think this would result in the form being filled with previous values even in case of success save. A redirect here in case of success would refresh everything.

Reply:
When I was testing this out, I kind of liked it since it would have the app/module filled out which would make it easier to fill out several additions at once. I will wait for GS feedback to decide if they do not want that